fixtures/typings-test/tsconfig.json

```diff
@@ -0,0 +1,12 @@
+{
+    "compilerOptions": {
+        "target": "es2017",
+        "module": "commonjs",
+        "noEmit": true
```

Comment:
Should we use this in the xterm repo as well?

Reply:
yeah i'll open a PR over there to flip this on and remove the unlink